lib/ruby_lsp/ruby_lsp_rails/server.rb

```diff
@@ -79,6 +81,33 @@ def execute(request, params)
 
       private
 
+      sig { params(name: String).returns(T::Hash[Symbol, T.untyped]) }
+      def route_location(name)
+        # Older versions of Rails don't support `route_source_locations`.
+        # We also check it hasn't been disabled.
+        unless ActionDispatch::Routing::Mapper.respond_to?(:route_source_locations) &&
```

Comment:
TODO: move into a conditionally defined method once the PR to remove Sorbet is merged.